requirements.pip

```diff
@@ -19,3 +19,4 @@ elasticsearch>=1.0.0,<2.0.0
 zipp>=1.2.0,<2
 # highest versions throw an error as has dropping python 2.7 support
 importlib-resources<=3.3.1
+contextlib2<0.7
```

Comment:
Should this be an `=` or maybe `<=` ?

Reply:
![Uploading Screenshot 2021-07-06 at 15.01.30.png…]()

Reply:
In case of any updates for version 0.6 I left an opportunity to apply it